sdk/cognitiveservices/textanalytics/Azure.AI.TextAnalytics/samples/Azure.AI.TextAnalytics.Samples.Tests.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+  </PropertyGroup>
+  <Import Project="..\..\..\..\core\Azure.Core\tests\LiveOnly.props" />
+  <ItemGroup>
+    <PackageReference Include="nunit" />
+    <PackageReference Include="NUnit3TestAdapter" />
```

Comment:
We have NUnit in some of our projects and XUnit in some other. Which one is preferred?

Reply:
For track 2 code, nunit.